packages/manifold/contracts/lazyclaim/ILazyPayableClaim.sol

```diff
@@ -42,6 +42,16 @@ interface ILazyPayableClaim {
      */
     function setMembershipAddress(address membershipAddress) external;
 
+    /**
+    *  @notice Set the mint fee for non-merkle claims
+    */
+    function setMintFee(uint256 mintFee) external;
```

Comment:
We should just have one function that takes in both.

Reply:
great point, just pushed the fix